tests/conftest.py

```diff
@@ -267,8 +267,9 @@ def reference_entry(exposure_factory, provenance_base, provenance_extra):
 
                 session.commit()
 
+
 @pytest.fixture
 def config_test():
     # Make sure the environment is set as expected for tests
-    assert os.getenv( "SEECHANGE_CONFIG" ) == "/seechange/tests/seechange_config_test.yaml"
+    # assert os.getenv( "SEECHANGE_CONFIG" ) == "/seechange/tests/seechange_config_test.yaml"
     return Config.get( os.getenv("SEECHANGE_CONFIG") )
```

Comment:
I don't understand why you removed this assert ; the tests are designed to go with that specific config file, so we want to make sure we've got the right config file before proceeding.